draft-ietf-mls-protocol.md

```diff
@@ -766,7 +766,9 @@ delete the leaf_secret.
 
 ## Synchronizing Views of the Tree
 
-The members of the group need to keep their views of the tree in
+After generating a Commit as described in the prior section, the generator of
+the Commit must broadcast this update to other members of the group, who will need to
+apply it to keep their views of the tree in
```

Comment:
Might rephrase as "apply it to their local views of the tree to keep them in sync with the sender's."